build/litmus-go/Dockerfile

```
@@ -23,6 +23,12 @@ RUN curl -L https://github.com/alexei-led/pumba/releases/download/${PUMBA_VERSIO
 #Installing promql cli binaries
 RUN curl -L https://github.com/litmuschaos/test-tools/raw/master/custom/promql-cli/promql-linux-${TARGETARCH} --output /usr/local/bin/promql && chmod +x /usr/local/bin/promql
 
+RUN if [[${TARGETARCH} == "arm64"]]; then \
```

```suggestion
RUN if [[ "${TARGETARCH}" == "arm64"]]; then \
```
We need to give gap and double quotes in bash.
ref: https://travis-ci.org/github/litmuschaos/litmus-go/builds/748600991#L1977

Also It will be good if we give `else if` for `amd64` than only else.